handlers/getItems.go

```diff
@@ -22,13 +22,13 @@ func newGetItems() RouteHandler {
 	}
 }
 
-func (g *getItems) match(request events.APIGatewayProxyRequest) bool {
+func (g *getItems) match(request events.APIGatewayV2HTTPRequest) bool {
```

Comment:
Most of this PR is moving from `APIGatewayProxyRequest` to `APIGatewayV2HTTPRequest`